.github/ISSUE_TEMPLATE/aws-access-request.yml

```diff
@@ -132,7 +132,7 @@ body:
       label: "User must exist in a roster before AWS access can be granted. See [AWS access documentation](https://vfs.atlassian.net/wiki/spaces/OT/pages/1792114735/Onboard+New+Team+Members+Granting+Access+to+Internal+Tools#AWS-Access-Request) for the steps involved."
       description: Don't fill this part out, please
       options:
-      - label: "Search for user on the [VFS Team Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)"
+      - label: "Search for user on the [VFS Team Roster](https://www.va.gov/atlas/product_directory/team_members)"
       - label: "Or search for user on the [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)"
       - label: "If user is on a VFS team but not in the VFS Team Roster, add the 'NOT YET' label and instruct them to start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)"
```

Comment:
Change "If user is on a VFS team but not in the VFS Team Roster" as well

Reply:
oh wait 🤔 I see you are keeping the name "VFS Roster" or "VFS Team Roster" and linking to Atlas here and everywhere else in the PR. Is that what we want or should we go all in and just call it "Atlas"?

Reply:
Search for user on Platform Atlas